packages/host/src/components/Footer/StoreSelector.tsx

```diff
@@ -34,6 +37,7 @@ export const StoreSelector: FC = ({ children }) => {
       onClick={() => {
         setStore(s);
         hide();
+        navigate(AppRoute.Dashboard);
       }}
```

Comment:
offroad.. when switching stores it would always tell me 'invalid invoice/requisition/etc' and I was sick of it.

Reply:
Yup me too, thanks.